src/emulator/gui/include/gui/state.h

```diff
@@ -90,6 +90,8 @@ struct State {
     char disassembly_count[5] = "100";
     std::vector<std::string> disassembly;
 
+    uint32_t background_texture = 0;
```

Comment:
Just wanted to point out that this isn't a special texture object or even a `GLuint`.